src/utils/types/userTypes.ts

```diff
@@ -0,0 +1,5 @@
+export enum UserType {
```

Comment:
can we change this to match the enum in user.label ["educator", "student", "parent", "administrator", "public"] so the prop passed in can be user.label directly? And then Header and Footer deal with the logic that educator/student/parent are the same